scenarios/kubernetes_build.py

```diff
@@ -162,6 +164,8 @@ def main(args):
         '--registry', help='Push images to the specified docker registry')
     PARSER.add_argument(
         '--extra-publish-file', help='Additional version file uploads to')
+    PARSER.add_argument(
+        '--extra-version-markers', help='Additional version file uploads to')
```

Comment:
Does this help need to be updated? Or should it really match `extra-publish-file`?

Reply:
It is serving the same purpose as `--extra-publish-file` (i.e. it will just overwrite `--extra-publish-file` if present), so it is likely fine to match unless we just want a new description altogether

Reply:
It is unclear to me how adding another flag with identical behavior will fix anything. I don't have the context to understand the motivation linked in the PR body though. Could you clarify what the problem is and how this fixes it? Why can't the callers of the script just use `--extra-publish-file`?

Reply:
@cjwagner adding the flag is not fixing anything, but it enables consumers to use the flag instead of `--extra-publish-file`, which has been deprecated in [push-build.sh](https://github.com/kubernetes/release/blob/b29efe949dfebf3ce4dbd277111924d089940bc3/push-build.sh#L49). Eventually I would expect `--extra-publish-file` to be removed.

Reply:
Got it. LGTM then.